src/components/Loader/index.jsx

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import "./Loader.css";
+const Loader = ({ isButtonLoader = false }) => {
+  return (
+    <div
+      className={`w-full ${
+        !isButtonLoader ? "h-5-6" : "h-auto"
+      } grid p-1 place-content-center`}
+    >
+      <div className="loader">
+        <div></div>
+        <div></div>
+        <div></div>
```

Comment:
Empty div tags?? May be you can use some library for loader

Reply:
Without empty divs It's hard to make a loader.